packages/constants/docs/LOG_LEVELS.md

```diff
@@ -0,0 +1,18 @@
+# Log Levels
+
+In order to promote consistant logging across the TV Kitchen ecosystem, the following log levels should be used as described when logging:
```

Comment:
```suggestion
In order to promote consistent logging across the TV Kitchen ecosystem, the following log levels should be used as described when logging:
```

Reply:
I really don't know how to spell consistent do I